### DISCUSSION TITLE
Can't get Fan Control to control GPU Fans

### DISCUSSION THREAD

**Discussion by BluestarP3D** [Rem0o/FanControl.Releases#727] [Q&A] [answered]
Setup Fan Control yesterday. Played some games for several hours, everything went well. However, I just noticed my fans were off a bit ago. Launched the app and noticed the temp was within my curve setting of 0 RPM when under 45*C. 

Launched a light game to push the temps up a bit. Watched it climb to 60*C and the fans never kicked on. 

Launched MSI Afterburner and it appears that it was controlling my fans. Kind of annoying considering I got Fan Control to control all my fans. I think I figured out how to disable the fan management, however I am not sure what is controlling the fan speed now. I have never set up fan controls in BIOS and I uninstalled the ASUS app that controlled my fans in the past (Xpert Fans). 

This leads me to believe that MSI Afterburner might still be controlling the fans. The only step I haven't tried is uninstalling MSI Afterburner, but trying to avoid that as I use it for testing undervolt settings and OSD.

Really lost and confused at this point

EDIT: I uninstalled MSI Afterburner and it didn't change anything. At this point, i'm going to assume that when I disable FanControl's controlling, that it's the standard GPU factory settings taking over or something. I don't have any other fan control applications intalled.
 
![explorer_Ur0HVGsq5R](https://user-images.githubusercontent.com/104739066/166166600-101fdad4-496b-4ded-8b57-d8bffa04315a.png)
![explorer_Y7VwMuVlil](https://user-images.githubusercontent.com/104739066/166166601-671c5762-adfe-4da0-972c-d29c871ba9af.png)

![image](https://user-images.githubusercontent.com/104739066/166167863-e4e13e0c-ee4b-4cb8-81f6-93256ffa045a.png)

**Comment by Fr0stX76** [comment, marked as answer]
First you should try to control them manually in FanControl to see if the control actually work.

I suspect your GPU bios has a minimum startup % around 50% (it is the case on my Strix 3090) So if you make a graph with a  fan % below this minimum start up percentage the fans will just not run.

Anyhow, see if the fan start in manual mode and if they do, note the startup % and make that the minimum % in your graph for the temp you want the fans to actually run.

**Reply by BluestarP3D** [reply]
I think that worked! Thank you! New curve is below. The only question now... What app do you use for OSD/Undervolting? MSI Afterburner was nice, but now that i've uninstalled it and lost all settings... Perhaps I might try something different?

Also, is there any plans on allowing the fans to shut off under a certain temp? I recall ASUS Fan Xpert doing this, which was nice.

P.S. Jealous of the Strix 3090... I can't believe the prices dropped so soon, spent $500 more than the cost of one for my TUF 3080 Ti OC and now out of my return date :(

![image](https://user-images.githubusercontent.com/104739066/166168368-e2327f58-8812-4e6a-a313-219eff597a40.png)

**Reply by Fr0stX76** [reply]
I use afterburner for undervolting. If I leave the fans on auto (in AB), it does not interfere with FanControl. 

As for stopping the fans at a certain temp, create a point 0.1 degree above the temp you want your fan to stop and another right next to it, at the temp you want them to stop but at 0%. You will also need to create a 0% point at 20 degrree.

So ex. if you want you fans to stop at  45C:

-Point 1, 20C  0%
-Point 2, 45C  0%
-Point 3, 45.1C 50%

Note that this will create a situation where if the temperature start going up and down when near the selected start up temp, the fans will go on and off often. To compensate, I would suggest incresing the response time to 5 or 10 seconds. I would also suggest ticking the "hysteresis only apply on the way down" checkbox

**Reply by BluestarP3D** [reply]
Thank you! I'll play with it a bit. I forgot to mention though, when I was looking for the minimum startup time, once it shut off, I was not able to trigger the fan to start back up, even when adjusting to 100%. I had to use the fan identifier, otherwise I had to relaunch the app.

Wondering if this trick will bypass whatever is causing that. Left the house after I got this sorted out in my last comment. Will check later on.

**Comment by kadiyakshitij** [comment]
I can't get it to show me controls at all. It detects there are 2 fans on my 6750xt but I can't seem to control them. I set it to a flat curve of 50% (the default one) but it just won't control the gpu fans.
Afterburner can control them with manual control when auto is off though

**Reply by Rem0o** [reply]
6750xt is not supported for controlling its fans.
